dev/PushNotifications/PushNotifications-Constants.h

```diff
@@ -12,9 +12,6 @@
 #define PUSHNOTIFICATIONS_IMPL_CLSID_UUID     E739C755-0D09-48DF-A468-A5DF0B5422DC
 #define PUSHNOTIFICATIONS_IMPL_CLSID_STRING   _STRINGIZE(PUSHNOTIFICATIONS_IMPL_CLSID_UUID)
 
-#define PUSHNOTIFICATIONS_TASK_CLSID_UUID     8FCFB82B-DB93-4E1D-9008-76D768CEB9EA
-#define PUSHNOTIFICATIONS_TASK_CLSID_STRING   _STRINGIZE(PUSHNOTIFICATIONS_TASK_CLSID_UUID)
```

Comment:
removing this will break the compilation. Please check PushNotificationManager.cpp

Reply:
That's true. I don't know how it compiled. But the variable is unused. I consulted this with @pmpurifoy and seems like we don't need them anymore.

Reply:
I just removed the reference in PushNotificationManager.